types/ExtrinsicSignature.ts

```diff
@@ -2,17 +2,33 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Address, Balance, Call, EcdsaSignature, Ed25519Signature, ExtrinsicEra, Index, MultiSignature, Sr25519Signature } from '@polkadot/types/interfaces/runtime';
-import { IExtrinsicSignature, IKeyringPair } from '@polkadot/types/types';
+import {
+    Address,
```

Comment:
Saw a lot of unnecessary changes which is not related to linting and formatting? Is it performed by linting tools?

Reply:
yes...Its formatted with tslint.. alphabetical ordering of imports and multiple imports can be combined into one (no duplicate imports) as specified in tslint